scripts/setup.go

```diff
@@ -175,18 +180,18 @@ func main() {
 		err = cluster.SetupWorkerKubelet(setupFlags.Args()[1])
 		// Original scripts from `scripts/cloudlab` directory
 	case "setup_node":
-		if setupFlags.NArg() < 2 {
-			utils.FatalPrintf("Missing parameters: %s <sandbox> [use-stargz]\n", subCmd)
+		if setupFlags.NArg() < 3 {
+			// ha_mode - REGULAR (do not install load balancers), MASTER, BACKUP (with load balancers)
+			utils.FatalPrintf("Missing parameters: %s <ha_mode> <sandbox> [use-stargz]\n", subCmd)
```

Comment:
We need to update the commands in the documentation with these parameters. And better to add the description of HA mode there.